config/gulp/translation.js

```diff
@@ -0,0 +1,71 @@
+var gulp = require('gulp');
+var gutil = require('gulp-util');
+var del = require('del');
+var pkg = require('../../package.json');
+var runSequence = require('run-sequence');
+var spawn = require('cross-spawn');
+var rename = require('gulp-rename');
+var replace = require('gulp-replace');
+var stateNames = require('./state-names.json');
```

Comment:
What do you think about renaming this file (and the variable it's loaded into) `es-state-names`/`esStateNames` to give more clarity around its purpose?